plugins/dfu/dfu-tool.c

```diff
@@ -918,10 +918,9 @@ dfu_tool_write (DfuToolPrivate *priv, gchar **values, GError **error)
 		if (!fu_device_detach (FU_DEVICE (device), error))
 			return FALSE;
 		if (!dfu_device_wait_for_replug (priv, device,
-						 FU_DEVICE_REMOVE_DELAY_RE_ENUMERATE,
-						 error)) {
+						 fu_device_get_remove_delay (device),
```

Comment:
Actually looks like this is failing CI: this is missing a cast of `FU_DEVICE (device)`

Reply:
Sorry about that. I will make the change.